ui/src/epics/view.ts

```diff
@@ -322,10 +320,30 @@ export const forceUpdateRowMeta: RootEpic = (action$, state$, { api }) =>
         })
     )
 
+const closeFormPopup: RootEpic = (action$, state$) =>
+    action$.pipe(
+        filter(actions.sendOperationSuccess.match),
```

Comment:
you should also add actions.bcSaveDataSuccess via isAnyOf

Reply:
Fixed